test/twofishes/client_test.rb

```diff
@@ -29,6 +29,90 @@ def self.geocode(location)
       assert result.parents
     end
 
+    it 'should accept max keyword argument' do
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', max: 5).first
+      assert result.parents
+    end
+
+    it 'should accept ll keyword argument' do
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
```

Comment:
You can use `Zurich` instead of `ZurichWithParents`. This should make it more clear that `ZurichWithParents` is a special version used to test the `ResponseIncludes`